tests/primitive/mad_hi.fut

```diff
@@ -0,0 +1,88 @@
+-- Test u8.mad_hi
+-- tags { no_csharp no_python}
```

Comment:
The `tags` field need to be after the `==` line.

Reply:
Fixed and fixed some test failures introduced when previously fixing parsing errors.